packages/ui-default/templates/partials/contest_user.html

```
@@ -10,7 +10,10 @@
     <td class="col--rank">{{ _('UnRank') if tsdoc.unrank else _('Rank') }}</td>
     <td class="col--actions">
       <a name="edit_rank" data-uid="{{ tsdoc.uid }}">{{ _('Rank') if tsdoc.unrank else _('UnRank') }}</a>
+      {% if tsdoc.endAt and tsdoc.endAt < tdoc.endAt %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)